Automattic/jetpack
# JS Components: fix Warning when defining AdminPage header prop

This PR defines the properly the type of the AdminPage header property.

Fixes #

#### Changes proposed in this Pull Request:

* JS Components: fix Warning when defining AdminPage header prop

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to Protect dashboard
* Without this PR, confirm you see the React warning about the prop type
<img width="815" alt="image" src="https://user-images.githubusercontent.com/77539/166577863-8638e11b-ed9d-46ba-ab0d-33a20493d398.png">
* With these changes, confirm the warning doesn't appear anymore
